File: 247drama-backend/test_db.py
import pymongo
from dotenv import dotenv_values

config = dotenv_values('.env')
client = pymongo.MongoClient(config['MONGODB_URL'])
db = client.get_default_database()
video = db.shortvideos.find_one()
if video:
    print('videoImage:', video.get('videoImage'))
else:
    print('no video')

File: 247drama-backend/test_db.js
const mongoose = require('mongoose');
const MovieSeries = require('./models/movieSeries.model');
const ShortVideo = require('./models/shortVideo.model');
require('dotenv').config({ path: './.env' });

mongoose.connect(process.env.MONGODB_URL, { useNewUrlParser: true, useUnifiedTopology: true })
  .then(async () => {
    const movie = await MovieSeries.findOne().lean();
    console.log('Movie thumbnail:', movie ? movie.thumbnail : 'none');
    const shortVideo = await ShortVideo.findOne().lean();
    console.log('ShortVideo videoImage:', shortVideo ? shortVideo.videoImage : 'none');
    process.exit(0);
  });
